### ISSUE TITLE
Cannot find -lMyLabel

### ISSUE THREAD

**Issue by Felix-neko** [eyllanesc/Qt5toPyQt5#1]
I've tried to do exactly as you wrote, but I got an error on `make` step:

```
make[1]: entering catalog «/home/felix/Projects/Qt5toPyQt5/src»
g++ -c -m64 -pipe -O2 -Wall -W -D_REENTRANT -fPIC -DANALOGCLOCK_LIBRARY -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o analogclock.o analogclock.cpp
/usr/lib/x86_64-linux-gnu/qt5/bin/moc -DANALOGCLOCK_LIBRARY -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -I/home/felix/Projects/Qt5toPyQt5/src -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/x86_64-linux-gnu/qt5/QtGui -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/include/c++/6 -I/usr/include/x86_64-linux-gnu/c++/6 -I/usr/include/c++/6/backward -I/usr/lib/gcc/x86_64-linux-gnu/6/include -I/usr/local/include -I/usr/lib/gcc/x86_64-linux-gnu/6/include-fixed -I/usr/include/x86_64-linux-gnu -I/usr/include analogclock.h -o moc_analogclock.cpp
g++ -c -m64 -pipe -O2 -Wall -W -D_REENTRANT -fPIC -DANALOGCLOCK_LIBRARY -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -I. -I/usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++-64 -o moc_analogclock.o moc_analogclock.cpp
rm -f libanalogclock.so.1.0.0 libanalogclock.so libanalogclock.so.1 libanalogclock.so.1.0
g++ -m64 -Wl,-O1 -shared -Wl,-soname,libanalogclock.so.1 -o libanalogclock.so.1.0.0 analogclock.o moc_analogclock.o  -L/usr/X11R6/lib64 -lQt5Widgets -lQt5Gui -lQt5Core -lGL -lpthread  
ln -s libanalogclock.so.1.0.0 libanalogclock.so
ln -s libanalogclock.so.1.0.0 libanalogclock.so.1
ln -s libanalogclock.so.1.0.0 libanalogclock.so.1.0
make[1]: выход из каталога «/home/felix/Projects/Qt5toPyQt5/src»
make[1]: вход в каталог «/home/felix/Projects/Qt5toPyQt5/modules»
g++ -c -g -O2 -fstack-protector-strong -Wformat -Werror=format-security  -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Wall -W -DNDEBUG -DMYLABEL_LIBRARY -DQT_CORE_LIB -DQT_GUI_LIB -DQT_WIDGETS_LIB -I. -I/home/felix/Projects/Qt5toPyQt5/src -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/include/x86_64-linux-gnu/qt5/QtGui -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/python3.5m -o sipPyAnalogClockcmodule.o sipPyAnalogClockcmodule.cpp
g++ -c -g -O2 -fstack-protector-strong -Wformat -Werror=format-security  -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Wall -W -DNDEBUG -DMYLABEL_LIBRARY -DQT_CORE_LIB -DQT_GUI_LIB -DQT_WIDGETS_LIB -I. -I/home/felix/Projects/Qt5toPyQt5/src -I/usr/include/x86_64-linux-gnu/qt5 -I/usr/include/x86_64-linux-gnu/qt5/QtCore -I/usr/include/x86_64-linux-gnu/qt5/QtGui -I/usr/include/x86_64-linux-gnu/qt5/QtWidgets -I/usr/include/python3.5m -o sipPyAnalogClockAnalogClock.o sipPyAnalogClockAnalogClock.cpp
g++ -Wl,-Bsymbolic-functions -Wl,-z,relro -shared -Wl,--version-script=PyAnalogClock.exp -o PyAnalogClock.so sipPyAnalogClockcmodule.o sipPyAnalogClockAnalogClock.o -L/usr/lib/x86_64-linux-gnu -L../src -L../src/release -lMyLabel -lQt5Core -lQt5Gui -lQt5Widgets
/usr/bin/ld: cannot find -lMyLabel
```

Can you give me an advice how to cope with it?

P.S. I'm using pyqt5-dev package of Ubuntu (16.10 x64)

**Comment by eyllanesc** [comment]
The problem is that I had left another project code. Already update the project. Please test it. Thank you for your reply.

**Comment by eyllanesc** [comment]
Follow these steps:

```
git clone https://github.com/eyllanesc/Qt5toPyQt5
cd Qt5toPyQt5
python configure.py
make 
sudo make install
cp modules/PyAnalogClock.so {Path of your script}
```

**Comment by Felix-neko** [comment]
Hi again!
I've reloaded the project. Now `make` step passes, but when I try to run Examples/main.py, it gives an error: ImportError: libMyLabel.so.1: cannot open shared object file: No such file or directory

**Comment by eyllanesc** [comment]
Have you replaced the PyAnalogClock.so file with the new one that was generated in the modules folder?

**Comment by eyllanesc** [comment]
By default, I left the .so file inside the folder of the example, this file deletes and copies the file that has been generated again and is in the modules folder, this file is called PyAnalogClock.so.

**Comment by Felix-neko** [comment]
Oh, it looks like the old PyAnalogClock.so Python module that was pre-added to `Examples` was calling the `libMyLabel`. I replaced it with new-generated PyAnalogClock.so, executed `sudo make install`, but when I ran `python3 Examples/main.py`, it crashed with `segmentation fault`...

**Comment by eyllanesc** [comment]
Do you run `sudo make install` or `sudo make installed`?

**Comment by eyllanesc** [comment]
To run `python configure` or `python3 configure`?

**Comment by Felix-neko** [comment]
Hi again!

I did `python3 configure.py` and `sudo make install`.


P.S. Thank you for patience, I hope we'll find the root of the problem.
^__^

**Comment by eyllanesc** [comment]
Can we connect by teamviewer?

**Comment by Felix-neko** [comment]
Oh, yes!
Just now I'm updating my Ubuntu to 17.04 version (maybe it will help). It will take about 15-20 munites -- and I'm installing the Teamviewer.

**Comment by eyllanesc** [comment]
Okay, I'll send the credentials to my mail: e.yllanescucho@gmail.com

**Comment by eyllanesc** [comment]
Run this please: `sudo apt-get install python3-pyqt5`

**Comment by Felix-neko** [comment]
Hola, comadre!

It looks stranger and stranger: the version of Qt installed on my Ubuntu now (`qmake --verson`) was 5.7.
And PyQt5 QT_VERSION (`from PyQt5.QtCore import QT_VERSION_STR`) was 5.8.

Maybe it was because I installed PyQt with `pip3 install`.

Then I typed `pip uninstall python3-pyqt5` and ran `sudo apt-get install python3-pyqt5`.  And PyQt5 now just fails with segfault when I'm just trying to `import PyQt5.QtGui`!

Anyway, I've sent you my TeamViewer credentials to your mail.
